test/OpenTelemetry.Instrumentation.SqlClient.Tests/OpenTelemetry.Instrumentation.SqlClient.Tests.csproj

```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.1" Condition="'$(TargetFramework)' != 'net452'" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="2.1.0" Condition="'$(TargetFramework)' != 'net452'" />
```

Comment:
Should we target 2.0.0? Don't want to force an unnecessary upgrade on anyone.

Reply:
This is test right?

Reply:
Yeah this is in the Tests csproj. The instrumentation itself doesn't reference it. For Tests I just selected to the latest stable on NuGet. Probably ok to keep?

Reply:
i mean - the actual project shouldn't depend on any package at all. For tests, we can test with latest, and it should be fine. Right?

Reply:
My bad! Fine to keep.